server/helpers/notifications.js

```diff
@@ -51,6 +56,16 @@ const NOTIFICATIONS = {
     type: NOTIFICATION_TYPE.SUCCESS,
     url: '/',
   },
+  OFFLINE_PAYMENT_ADDED: {
+    description: 'Your offline payment has been added successfully',
```

Comment:
Can we start updating the description to give the right message. I think it is better to fix them now than waiting to fix them in another PR.
```suggestion
    description: 'You added an offline Payment of N {payment} for `{property}`',
```